### PR TITLE
Wazuh Cloud: Documentation for the new pricing model

### PR DESCRIPTION
**Pull request by jesuslinares**
This is a refactoring of the current documentation to include the new pricing model (unreleased).

Thanks.

### CODE REVIEW

**Review by Spades0 — COMMENTED** [review]
I left a few suggestions, LGTM after addressing them

**Comment by jesuslinares** [comment]
Everything is ready to be merged, but we need to wait until the new pricing model is released before we can publish it.

File: source/cloud-service/getting-started/sign-up-trial.rst [hunk 1]
```diff
@@ -26,26 +29,24 @@ Create environment
...
+   #. Fill in the use case. This information helps us understand why our users utilize our service, allowing us to improve it accordingly.
 
-   #. Choose the support plan that best suits your needs. 
-
-   #. Complete your user profile. This step is only shown when there is missing information on your profile. 
+   #. Select the desired settings. To simplify the process, we have prepared three predefined sets of settings called profiles: Small, Medium, and Large. If none of these profiles meet your requirements, you can click Custom and configure your settings individually. To learn more about the settings and their functionality, see the :ref:`Settings <cloud_your_environment_settings>` section.
    
-      To edit your user profile information, see the :ref:`Edit user settings <cloud_user_settings>` section.
+      .. note:: During the 14-day trial period, some settings are limited, but they do not prevent you from gaining insight into the Wazuh Cloud platform. All restrictions are removed after you make the first payment.
```
Comment (javimed) [inline source/cloud-service/getting-started/sign-up-trial.rst:44]:
```suggestion
      .. note:: Thoughout the 14-day trial period, some settings are limited. However, they do not prevent you from exploring and using the Wazuh Cloud platform. All restrictions are removed after you make the initial payment.
```

File: source/cloud-service/getting-started/sign-up-trial.rst [hunk 2]
```diff
@@ -26,26 +29,24 @@ Create environment
...
+
+   #. Select between monthly or annual payments. If you choose the monthly option, you will be billed on a monthly basis, whereas the annual option entails a single payment per year.
 
-#. Click **View the summary** and then **Create** to build your environment. This process might take a moment.
+#. Click on **Start your free trial** to build your environment. This process might take a few minutes.
 
-Once the environment is ready, you can :ref:`access the Wazuh WUI <cloud_wui_access>`  and register the agents. 
+Once the environment is ready, access the :ref:`Wazuh WUI <cloud_wui_access>` and enroll the Wazuh agents.
```
Comment (javimed) [inline source/cloud-service/getting-started/sign-up-trial.rst:50]:
```suggestion
Once the environment is ready, access the :ref:`Wazuh dashboard <cloud_wui_access>` and enroll the Wazuh agents.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 3]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
-- `How can I get help?`_
+- `What happens if the data ingestion rate is exceeded?`_
+
+- `Can I cancel at any time?`_
   
 What is Wazuh Cloud?
 --------------------
 
-Wazuh Cloud hosts and manages all the Wazuh components in one integrated platform, you simply need to access your environment and deploy the Wazuh agent to your endpoints. Wazuh provides protection with *Security Information and Event Management* (SIEM) and *Endpoint Detection and Response* (EDR).
+Wazuh Cloud provides a comprehensive solution by hosting and managing all Wazuh components within a single integrated platform. With just a few clicks, you can effortlessly create your environment and enroll Wazuh agents. By leveraging Wazuh, you gain access to powerful security capabilities, including Security Information and Event Management (SIEM) and Extended Detection and Response (XDR).
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:53]:
```suggestion
Wazuh Cloud provides a comprehensive solution by hosting and managing all Wazuh components within a single integrated platform. Wazuh cloud allows you to create your environment and enroll Wazuh agents. By leveraging Wazuh, you gain access to security capabilities, including Security Information and Event Management (SIEM) and Extended Detection and Response (XDR).
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 4]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+Yes, the Wazuh Cloud platform is compliant with SOC 2 standards.
+
 .. _cloud_getting-started_support:
 
 How can I get support?
 ----------------------
 
-Every environment includes support after the first payment.
+Every environment includes support after the first payment. You can contact the Wazuh team anytime through the Help section on your Wazuh Cloud Console. You can also fill out `this form <https://wazuh.com/cloud/#subscription>`_ to get help from the Wazuh team.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:82]:
```suggestion
Every environment includes support after the initial payment. You can contact the Wazuh team anytime through the Help section on your Wazuh Cloud Console. You can also fill out `this form <https://wazuh.com/cloud/#subscription>`_ to get help from the Wazuh team.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 5]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
 
 Where is Wazuh Cloud hosted?
 ----------------------------
 
 Wazuh Cloud is hosted on Amazon Web Services (AWS).
 
+What is a profile?
+------------------
+
+A profile refers to predefined settings that you can choose from when configuring your Wazuh Cloud environment. We have three profiles available: Small, Medium, and Large. These profiles provide ready-to-use environment templates that cater to different needs and requirements. If none of the predefined profiles meet your specific requirements, you can configure your settings individually.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:92]:
```suggestion
A profile refers to predefined settings available for configuring your Wazuh Cloud environment. We offer three profiles: Small, Medium, and Large. They provide ready-to-use environment templates that cater to different needs and requirements. If none of these predefined profiles meet your specific requirements, you can configure your settings individually.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 6]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+What is a setting?
+------------------
+
+A setting in the context of Wazuh Cloud refers to each configuration option available for a cloud environment. These settings determine the limitations and capabilities of an environment. For example, the "Active Agents" setting specifies the maximum number of active agents allowed in your environment. It is important to note that the settings chosen for an environment directly impact its pricing, as they affect the resources allocated and functionalities available.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:97]:
```suggestion
In the context of Wazuh Cloud, a setting  refers to each configuration option available for a cloud environment. These settings determine the limitations and functionalities of an environment. For example, the "Active Agents" setting specifies the maximum number of active agents allowed in your environment. Note that the settings chosen for an environment directly impact its pricing, as they affect the resources allocated and functionalities available.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 7]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+What is the indexed data?
+-------------------------
+
+The indexed data, previously known as hot storage, includes the data available on the Wazuh WUI, which corresponds to the information indexed by Wazuh. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:102]:
```suggestion
The indexed data, previously known as hot storage, includes the data available on the Wazuh dashboard, which corresponds to the information Wazuh indexed. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 8]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+What is the archived data?
+--------------------------
+
+This Wazuh generated data is archived in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this data is not searchable or analyzable. It simply consists of a collection of compressed files. For more information, please see the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:107]:
```suggestion
The data generated by Wazuh is archived in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this archived data is not searchable or analyzable. It simply consists of a collection of compressed files. For more information, please see the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 9]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
 What is a tier?
 ---------------
 
-The tier is the size limitation, in bytes, of the hot storage.
+The concept of a tier, which represents the size limitation, in bytes, of the indexed data (formerly known as hot storage), is no longer used. It has been replaced by the *indexed data size* setting.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:112]:
```suggestion
The term *tier* is no longer used. It has been replaced by the *indexed data size* setting. It represented the size limitation, in bytes, of the indexed data, formerly known as *hot storage*.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 10]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
 
-The oldest events are removed from your index when the tier limit is reached. This data is available in cold storage for you to access. See the :ref:`Cold storage <cloud_your_environment_accessing_cold_storage>` section to learn more about data logging and storage.
+See :ref:`What happens if the indexed data size setting is reached? <cloud_starting_faq_data_reached>`.
+
+.. _cloud_starting_faq_data_reached:
+
+What happens if the indexed data size setting is reached?
+---------------------------------------------------------
+
+When the selected **indexed data size** is reached, the oldest events will be automatically removed from your index regardless of the **index data time**. This data is available in archived data for you to access. See the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section to learn more about data logging and storage.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:124]:
```suggestion
When the selected **indexed data size** is reached, Wazuh automatically removes the oldest events from your index, regardless of the **index data time**. You can access this data in archived data. See the :ref:`Archived data <cloud_your_environment_accessing_archived_data>` section to learn more about data logging and storage.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 11]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+How is indexed data rotated?
+----------------------------
+
+Data rotation is determined by two conditions: the indexed data time and the indexed data size. For instance, if you have set a 3-month indexed data time and a 100GB indexed data size, and you consume the entire 100GB within the first month, the data will start rotating. Similarly, even if you have not utilized all 100GB, the data from the first month will be rotated by the fourth month if only 20GB is used.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:129]:
```suggestion
Data rotation is determined by two conditions:

-  Indexed data time
-  Indexed data size

For example, you might have set a 3-month indexed data time and a 100 GB indexed data size. If you consumed the entire 100 GB within the first month, the data starts rotating.

Similarly, you might have not utilized all 100 GB. However, the data from the first month rotates by the fourth month even if only 20 GB is used.
```

File: source/cloud-service/getting-started/starting-faq.rst [hunk 12]
```diff
@@ -21,67 +21,119 @@ Cloud service FAQ
...
+
+What happens if the data ingestion rate is exceeded?
+----------------------------------------------------
+
+If you send more events per second than allowed by the **data ingestion rate** setting, events start queuing, and if the queue becomes full, the incoming events will be discarded, which may lead to potential event loss.
```
Comment (javimed) [inline source/cloud-service/getting-started/starting-faq.rst:134]:
```suggestion
If you send more events per second than allowed by the **data ingestion rate** setting, events start queuing, and if the queue becomes full, the incoming events get discarded, which might lead to potential event loss.
```

File: source/cloud-service/glossary.rst [hunk 13]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
 .. _cloud_glossary_environment:
 
 Environment
 -----------
 
 An environment is a deployment that contains all the Wazuh components ready to use and running on Wazuh Cloud.
 
+.. _cloud_glossary_archived_data:
+
+Archived data (formerly known as cold storage)
```
Comment (javimed) [inline source/cloud-service/glossary.rst:58]:
```suggestion
Archived data
```

File: source/cloud-service/glossary.rst [hunk 14]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
+----------------------------------------------
+
+It is the data containing the output generated by Wazuh, such as alerts or archives. It is an AWS S3 bucket to store your logs for a longer time and meet compliance requirements.
```
Comment (javimed) [inline source/cloud-service/glossary.rst:61]:
```suggestion
Formerly known as cold storage, it's the data containing the output generated by Wazuh, such as alerts and archives. It's an AWS S3 bucket to store your logs for a longer time and meet compliance requirements.
```

File: source/cloud-service/glossary.rst [hunk 15]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
+
```
Comment (javimed) [inline source/cloud-service/glossary.rst:62]:
```suggestion
```

File: source/cloud-service/glossary.rst [hunk 16]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
+
 .. _cloud_glossary_hot_storage:
 
-Hot storage
------------
+Indexed data (formerly known as hot storage)
```
Comment (javimed) [inline source/cloud-service/glossary.rst:66]:
```suggestion
Indexed data
```

File: source/cloud-service/glossary.rst [hunk 17]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
+
+
 .. _cloud_glossary_hot_storage:
 
-Hot storage
------------
+Indexed data (formerly known as hot storage) 
+--------------------------------------------
 
 It is the data available on the Wazuh WUI corresponding to the information indexed by Wazuh. This information is available as soon as Wazuh ingests and indexes the events sent by the agents, making the data searchable and analyzable.
```
Comment (javimed) [inline source/cloud-service/glossary.rst:69]:
```suggestion
Formerly known as hot storage, it's the data available on the Wazuh dashboard corresponding to the information indexed by Wazuh. This information is available as soon as Wazuh ingests and indexes the events sent by the agents, making the data searchable and analyzable.
```

File: source/cloud-service/glossary.rst [hunk 18]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
 
-Hot storage use is calculated using the primary shards of ``wazuh-*`` indices.
+Indexed data is calculated using the primary shards of ``wazuh-*`` indices.
+
+.. _cloud_glossary_tier:
+
+Tier
+----
+
+The concept of a tier, which represents the size limitation, in bytes, of the indexed data (formerly known as hot storage), is no longer used. It has been replaced by the *indexed data size* setting.
```
Comment (javimed) [inline source/cloud-service/glossary.rst:78]:
```suggestion
The term *tier* is no longer used. It has been replaced by the *indexed data size* setting. It represented the size limitation, in bytes, of the indexed data, formerly known as *hot storage*.
```

File: source/cloud-service/glossary.rst [hunk 19]
```diff
@@ -41,28 +46,50 @@ Cloud ID
...
+
+.. _cloud_glossary_setting:
+
+Setting
+-------
+
+A setting in the context of Wazuh Cloud refers to each of the configuration options available for a cloud environment. These settings define the limitations, capabilities, and pricing of the environment.
```
Comment (javimed) [inline source/cloud-service/glossary.rst:85]:
```suggestion
In the context of Wazuh Cloud, a setting  refers to each configuration option available for a cloud environment. These settings determine the limitations, functionalities, and pricing of an environment.
```

**Review by javimed — CHANGES_REQUESTED** [review]
I think this documents update is meant to be out for 4.7. Therefore, the PR base branch must be changed to `4.7`.

Also, please consider the suggestions left so far.

File: source/cloud-service/your-environment/settings.rst [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+----------------------------------
+
+Below, we provide a detailed explanation of each setting.
+
+.. _cloud_settings_active_agents:
+
+Active agents
+^^^^^^^^^^^^^
+
+This basic setting defines the maximum number of active Wazuh agents that the environment can support. Please note that while it is possible to register an unlimited number of Wazuh agents, the number of active agents is limited by this setting.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:25]:
```suggestion
This basic setting sets the maximum count of active Wazuh agents that the environment can support. Please note that while registering an unlimited number of Wazuh agents is possible, the active agent count is limited by this setting.
```

File: source/cloud-service/your-environment/settings.rst [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+
+If the maximum number of active agents is reached, the environment may start to malfunction, causing instability with agent connections. While the system can tolerate temporarily exceeding the limit of active agents, appropriate measures will be taken if the situation persists.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:27]:
```suggestion
If the maximum number of active agents is reached, the environment might start to malfunction, causing instability with agent connections. Although the system can temporarily handle exceeding the active agent limit, appropriate measures will be taken if the situation persists.
```

File: source/cloud-service/your-environment/settings.rst [hunk 22]
```diff
@@ -0,0 +1,95 @@
...
+
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:28]:
```suggestion
```

File: source/cloud-service/your-environment/settings.rst [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+
+.. _cloud_settings_indexed_data:
+
+Indexed data
+^^^^^^^^^^^^
+
+The indexed data, previously known as hot storage, includes the data available on the Wazuh WUI, which corresponds to the information indexed by Wazuh. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:35]:
```suggestion
The indexed data was previously known as hot storage. It includes the data available on the Wazuh dashboard, which corresponds to the information indexed by Wazuh. This information becomes searchable and analyzable as soon as Wazuh ingests and indexes the events sent by the agents.
```

File: source/cloud-service/your-environment/settings.rst [hunk 24]
```diff
@@ -0,0 +1,95 @@
...
+
+Two settings define the behavior of the indexed data:
+
+- Indexed data time: It determines the maximum duration for which data remains indexed. This is a basic setting.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:39]:
```suggestion
-  **Indexed data time**: This basic setting defines the maximum duration for which data remains indexed.
```

File: source/cloud-service/your-environment/settings.rst [hunk 25]
```diff
@@ -0,0 +1,95 @@
...
+
+- Indexed data size: It defines the maximum size, in bytes, of the indexed data. This setting is equivalent to what was previously called a "tier." This is an advanced setting, and the interface provides a suggestion when selecting the Indexed data time.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:41]:
```suggestion
-  **Indexed data size**: This advanced setting defines the maximum size, in bytes, of the indexed data. it was formerly referred to as a "tier". The interface offers a suggestion when selecting the Indexed data time.
```

Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:41]: It's not quite clear what the "... suggestion when selecting the Indexed data time." is all about.

File: source/cloud-service/your-environment/settings.rst [hunk 26]
```diff
@@ -0,0 +1,95 @@
...
+
+Data remains indexed until either the indexed data time or the indexed data size is reached. In other words, once either of the settings' values is reached, data rotation will occur (removing the oldest data) until the settings' conditions are met.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:43]:
```suggestion
Data remains indexed until either the indexed data time or the indexed data size is reached. Upon reaching either setting value, data rotation initiates, removing the oldest data, until the conditions of these settings are met.
```

File: source/cloud-service/your-environment/settings.rst [hunk 27]
```diff
@@ -0,0 +1,95 @@
...
+
+
+.. _cloud_settings_archived_data:
+
+Archived data
+^^^^^^^^^^^^^
+
+This basic setting, previously known as cold storage, determines the duration for which the analyzed data generated by Wazuh is stored in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this data is not searchable or analyzable. It simply consists of a collection of compressed files.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:51]:
```suggestion
This basic setting, previously known as cold storage, defines the duration for which the analyzed data generated by Wazuh is stored in an AWS S3 bucket for long-term storage purposes. Unlike the indexed data, this data isn't searchable or analyzable. It simply consists of a collection of compressed files.
```

File: source/cloud-service/your-environment/settings.rst [hunk 28]
```diff
@@ -0,0 +1,95 @@
...
+
+When the specified time is reached, data outside that time range will be deleted.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:53]:
```suggestion
When the specified time is reached, data beyond that time range will be deleted.
```

File: source/cloud-service/your-environment/settings.rst [hunk 29]
```diff
@@ -0,0 +1,95 @@
...
+^^^^^^^^^^^^
+
+This setting indicates whether the support level is premium or standard.
+
+.. _cloud_settings_data_ingestion_rate:
+
+Data ingestion rate
+^^^^^^^^^^^^^^^^^^^
+
+It represents the maximum number of events per second (EPS) that the environment can analyze. This is an advanced setting, and the interface provides a suggestion when selecting the Active agents setting.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:67]:
```suggestion
It represents the maximum number of events per second (EPS) that the environment can analyze. This is an advanced setting, and the interface provides a suggestion when selecting the ``Active agents`` setting.
```

File: source/cloud-service/your-environment/settings.rst [hunk 30]
```diff
@@ -0,0 +1,95 @@
...
+
+If the ingestion rate is exceeded, events will start to queue. However, if the queue becomes full, the incoming events will be discarded, which may lead to potential event loss. The queuing mechanism is automatically managed by the cloud service, ensuring optimal resource utilization.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:69]:
```suggestion
If the ingestion rate is exceeded, events begin queuing. However, if the queue becomes full, wazuh discards the incoming events, which might lead to event loss. The cloud service automatically manages the queuing mechanism, ensuring optimal resource utilization.
```

File: source/cloud-service/your-environment/settings.rst [hunk 31]
```diff
@@ -0,0 +1,95 @@
...
+
+The environment will be configured with the `limits eps option <https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#limits>`_ using the following parameters:
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:71]:
```suggestion
The environment is configured with the `limits eps option <https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/global.html#limits>`_ using the following parameters:
```

File: source/cloud-service/your-environment/settings.rst [hunk 32]
```diff
@@ -0,0 +1,95 @@
...
+
+- timeframe = 1 seconds
+- maximum = Data ingestion rate setting / number of server nodes
+
+The number of server nodes is automatically determined by the cloud service based on the workload. For instance, if the Data ingestion rate setting is 900 EPS and there is a cluster of 2 nodes at the current time, each node can process up to 450 events per second.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:76]:
```suggestion
The cloud service automatically determines the number of server nodes  based on the workload. For instance, if the ``Data ingestion rate`` setting is ``900`` EPS and the current cluster consists of 2 nodes, each node can process up to 450 events per second.
```

File: source/cloud-service/your-environment/settings.rst [hunk 33]
```diff
@@ -0,0 +1,95 @@
...
+
+.. _cloud_settings_adjust:
+
+Adjusting environment settings
+------------------------------
+
+Managing your environment settings is crucial to meeting your evolving needs and optimizing the performance of your cloud environment. While some settings can be determined upfront, such as the number of active agents, indexed data time, archived data, and support type, it's important to note that these requirements may change over time.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:83]:
```suggestion
Managing your environment settings is crucial to meet your evolving needs and optimize the performance of your cloud environment. While some settings can be determined upfront, such as the number of active agents, indexed data time, archived data, and support type, note that these requirements might change over time.
```

File: source/cloud-service/your-environment/settings.rst [hunk 34]
```diff
@@ -0,0 +1,95 @@
...
+
+Additionally, advanced settings may be more challenging to determine in advance. While the interface provides recommendations based on our experience, your specific workload may differ. That's why we highly recommend deploying, monitoring, and adjusting the settings as needed to align with your evolving requirements.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:85]:
```suggestion
Advanced settings might be more challenging to determine in advance. While the interface provides recommendations based on our experience, your specific workload might differ. Hence, we recommend deploying, monitoring, and adjusting the settings as needed to align with your evolving requirements.
```

File: source/cloud-service/your-environment/settings.rst [hunk 35]
```diff
@@ -0,0 +1,95 @@
...
+
+To effectively monitor and adapt your environment, you have the option to modify your settings by opening a support ticket. Here's how the process works:
+
+- Upgrading a setting: If you need to raise a setting, you will be charged a prorated amount based on the remaining time in your billing cycle. The change will be implemented immediately after the payment is made. Please note that your next billing cycle will reflect the increased cost of the enhancement.
```
Comment (javimed) [inline source/cloud-service/your-environment/settings.rst:89]:
```suggestion
-  **Upgrading a setting**: If you need to raise a setting, you will be charged a prorated amount based on the remaining time in your billing cycle. The change will be implemented immediately after the payment is made. Please note that your next billing cycle will reflect the increased cost of the enhancement.
```

**Review by javimed — CHANGES_REQUESTED** [review]
The `source/_static/js/redirects.js` file needs to be updated to reflect the new and removed pages. Please consider the additional suggestions from the latest review as well.

File: source/cloud-service/your-environment/technical-faq.rst [hunk 36]
```diff
@@ -51,17 +59,43 @@ It is possible to get a new URL by opening a support ticket through the **Help**
...
+
+What if I need to upgrade or downgrade a setting?
+-------------------------------------------------
+
+You can upgrade or downgrade a setting by contacting the Wazuh team through the **Help** section of your Wazuh Cloud Console. See also :ref:`Adjusting environment settings <cloud_settings_adjust>`.
+
+What happens if active agents setting is reached?
+-------------------------------------------------
+
+If the maximum number of active agents is reached, the environment may start to malfunction, causing instability with agent connections. While the system can tolerate temporarily exceeding the limit of active agents, appropriate measures will be taken if the situation persists.
```
[the remark this answers is not in the document]
Comment (jesuslinares) [reply source/cloud-service/your-environment/technical-faq.rst:92]: We will leave that undefined.

File: source/cloud-service/archived-data/configuration.rst [hunk 37]
```diff
@@ -0,0 +1,26 @@
...
+
+- The file ``/var/ossec/logs/alerts/alerts.json`` contains only events that tripped a rule with high enough priority, according to a configurable threshold. This is always sent to archived data.
+
+Both files are delivered to archived data as soon as they are rotated and compressed. This process usually takes between 10 and 30 minutes from the moment the event is received.
+
+The oldest files in the archived data are rotated based on the **archived data** setting.
+
+.. note::
+
+  Files with a ``.log`` extension are never sent to archived data.
```
[the remark this answers is not in the document]
Comment (jesuslinares) [reply source/cloud-service/archived-data/configuration.rst:25]: It is just a generic way to say it.

**Comment by jesuslinares** [comment]
Suggestion are reviewed and 4.7 branch is merged.

Thanks @javimed!

**Pull request merged**